.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+update_configs:
+  - package-ecosystem: "gradle"
+    directory: "AccessibilityInsightsForAndroidService/app/"
+    default_labels:
```

Comment:
https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates suggests that there are a few breaking format changes between v1 and v2. In particular, it looks like:

* `directory` should start with `/`
* `default_labels` is now just `labels`
* `commit_message` is now `commit-message`
* `include_scope: true` is now `include: "scope"`